### PR TITLE
fix spend detail: wrong check ready

### PR DESCRIPTION
**Pull request by edouardparis**

### CODE REVIEW

**Review by edouardparis — COMMENTED** [review]
I feel bad

File: gui/src/app/view/spend/detail.rs [hunk 1]
```diff
@@ -392,7 +392,7 @@ pub fn signatures<'a>(
                     Container::new(
                         Row::new()
                             .push(Space::with_width(Length::Fill))
-                            .push_maybe(if tx.path_ready().is_some() {
+                            .push_maybe(if !tx.path_ready().is_some() {
```
Comment (darosior) [inline gui/src/app/view/spend/detail.rs:395]: nit (and i think clippy would complain)
```suggestion
                            .push_maybe(if tx.path_ready().is_none() {
```

**Comment by darosior** [comment]
Works for me now. Ack.

**Review by edouardparis — COMMENTED** [review]
Self-ACK 4c683993bc32616c0506528dec6f497f32f7c933

**Pull request merged**